vmlinux_to_elf/kallsyms_finder.py

```diff
@@ -1069,15 +1069,15 @@ def print_symbols_debug(self):
             
             symbol_types.add(symbol_name[0])
         
-        print('Symbol types', '=>', sorted(symbol_types))
-        print()
+        logging.info('Symbol types => {0:s}'.format(sorted(symbol_types)))
```

Comment:
It seems that you're passing a `list` to the `{0:s}` formatter, it's yielding an error when using the `kallsyms-finder` entry point:

```
  File "./kallsyms-finder", line 1072, in print_symbols_debug
    logging.info('Symbol types => {0:s}'.format(sorted(symbol_types)))
TypeError: unsupported format string passed to list.__format__
```

(Maybe that using `'%s' % value` for formatting is safer)